lib/money_column/active_record_hooks.rb

```diff
@@ -7,25 +18,7 @@ def self.included(base)
     module ClassMethods
       def money_column(*columns)
```

Comment:
I think I'll merge as is for now, because it seems the entire gem is deprecated anyway.